src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipCompletedFunction.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Azure.WebJobs;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.NServiceBus.AzureFunction.Attributes;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Forecasting.Commitments.Functions.NServicebusTriggerFunctions
+{
+    public class ApprenticeshipCompletedFunction
+    {
+        private readonly IForecastingDbContext _forecastingDbContext;
+
+        public ApprenticeshipCompletedFunction(IForecastingDbContext forecastingDbContext)
+        {
+            _forecastingDbContext = forecastingDbContext;
+        }
+
+        [FunctionName("ApprenticeshipCompleted")]
+        public async Task Run(
+            [NServiceBusTrigger(Endpoint = "SFA.DAS.Fcast.ApprenticeshipCompletedEvent")] ApprenticeshipCompletedEvent message)
+        {
+            var selectedApprenticeship = _forecastingDbContext.Commitment.Where(x => x.ApprenticeshipId == message.ApprenticeshipId).First();
```

Comment:
How do we know first is the only/correct apprenticeship? If we know there is only 1 then we should use Single.

Also what do we do if the apprenticeship does not exist for this event?

Reply:
Made changes,  CON-2700 contains further changes for no apprenticeship found.